spring-cloud-common-security-config/spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/CommonSecurityAutoConfiguration.java

```diff
@@ -21,14 +21,12 @@
 import org.springframework.boot.autoconfigure.security.oauth2.resource.servlet.OAuth2ResourceServerAutoConfiguration;
 import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Import;
 
 @Configuration(proxyBeanMethods = false)
 @AutoConfigureBefore({
 		SecurityAutoConfiguration.class,
 		ManagementWebSecurityAutoConfiguration.class,
 		OAuth2ClientAutoConfiguration.class,
 		OAuth2ResourceServerAutoConfiguration.class})
-@Import({IgnoreAllSecurityConfiguration.class, OAuthSecurityConfiguration.class})
```

Comment:
The import for `IgnoreAllSecurityConfiguration` and `OAuthSecurityConfiguration` have been removed here. For Dataflow server this has been replaced by the direct import of `SecurityConfiguration` in `DataflowServerConfiguration`. Does the Skipper server have a replacement?

Reply:
I think this is one of those difference between dataflow/skipper. There's already comments about this same line above.